server/ColorDeconvolution/ColorDeconvolution.xml

```diff
@@ -60,5 +60,37 @@
       <longflag>stainColor_3</longflag>
       <default>0,0,0</default>
     </double-vector>
+    <region>
+      <name>region</name>
+      <label>Region</label>
+      <longflag>region</longflag>
+      <description>left,top,width,height of the region of interest.  All -1 means the whole image is used.</description>
+      <default>-1,-1,-1,-1</default>
+    </region>
+    <!-- And now the output parameters -->
+    <region>
+      <name>region</name>
+      <label>Region</label>
+      <channel>output</channel>
+      <description>left,top,width,height of the region of interest.  All -1 means the whole image was used.</description>
+    </region>
+    <string>
```

Comment:
Yes, lets use image parameters and drop the returnparameterfile for now until we figure what would be convenient on the front-end side.